src/vaibhav/Assignment_17/EvenOddCount.java

```diff
@@ -0,0 +1,34 @@
+/*Java Assignment 17: 10th April 2021
+
+Program 1: From given numbers, count the odd and even numbers.
+
+input : 10,23,26,27,88,87,65
+output : even -> 3
+odd -> array.length - evenCount
+*/
+
+package vaibhav.Assignment_17;
+
+public class EvenOddCount {
+
+	int[] number = new int[8];
+
+	int evenCount;
+
+	void toFindEvenOddCount(int[] number) {
+		for (int index = 0; index < number.length; index++) {
+			if (number[index] % 2 == 0)
+				evenCount++;
+		}
+		System.out.println("Count for Even Number form given set of numbers is : " + evenCount);
+		System.out.println("Count for Odd Number form given set of numbers is  : " + (number.length - evenCount));
```

Comment:
smart way @VaibhavTati 
keep addressing this kind points

Reply:
Thank You @harshvegada